src/amount.h

```diff
@@ -15,4 +15,16 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
+/** No amount larger than this (in satoshi) is valid.
+ *
+ * Note that this constant is *not* the total money supply, which in Bitcoin
+ * currently happens to be less than 21,000,000 PIV for various reasons, but
+ * rather a sanity check. As this sanity check is used by consensus-critical
+ * validation code, the exact value of the MAX_MONEY constant is consensus
+ * critical; in unusual circumstances like a overflow bug that allowed
+ * for the creation of coins out of thin air modification could lead to a fork.
```

Comment:
I think this comment block should better reflect PIVX. fe: PIVX never experienced the overflow bug mentioned here, and the comment references Bitcoin's total money supply, which has no bearing for us.